### ISSUE TITLE
Unable to locate Eye of the Storm planet with Warp Controller

### ISSUE THREAD

**Issue by Deathlark** [FTBTeam/FTB-Continuum#247]
* **Modpack Version**: 1.3.1

* **Issue**: Unable to locate Eye of the Storm planet due to a typo in planetDefs.xml file(?)

* **Mod/s Affected**: Advanced Rocketry

* **Known Fix**: Delete a specific portion from a code in order to re-enable(?)

This is more or less a report on a typo that prevents progression by making it impossible to locate the Eye of the Storm planet. I've pasted the portion of the planetDefs.xml file from the 1.3.1 config relevant to this, and my suspicion is that the line "<artifact>minecraft:coal 1</artifact>" should be "<artifact>minecraft:coal</artifact>" given how modded items and vanilla items handle for identity, leading to it not working. As for testing this myself, I have attempted to locate the planet over 25 times and had no success using the listed artifacts, leading to my belief that this is the issue, although I have not tested this solution myself.

```
-<planet name="The Eye of the Storm" customIcon="marslike" DIMID="45">

<isKnown>false</isKnown>

<fogColor>1.0,1.0,1.0</fogColor>

<skyColor>0.964,0.392,0.074</skyColor>

<gravitationalMultiplier>100</gravitationalMultiplier>

<orbitalDistance>10</orbitalDistance>

<orbitalTheta>0</orbitalTheta>

<orbitalPhi>180</orbitalPhi>

<rotationalPeriod>24000</rotationalPeriod>

<atmosphereDensity>200</atmosphereDensity>

<biomeIds>advancedrocketry:stormland</biomeIds>

<oceanBlock>minecraft:lava</oceanBlock>

<artifact>minecraft:coal 1</artifact>

<artifact>minecraft:fire_charge</artifact>

<artifact>fluxnetworks:flux</artifact>

<artifact>tconstruct:firewood</artifact>

<artifact>quark:charred_nether_bricks</artifact>

</planet>


```

**Comment by TechnicianLP** [comment]
DId you try charcoal instead of regular coal?

**Comment by aaronhowser1** [comment]
Yep, it should be Charcoal, which is minecraft:coal 1. That was a mistake in the quest that's already been fixed in the next version (see #219)
